daml/splice-amulet/daml/Splice/Issuance.daml

```diff
@@ -57,11 +57,17 @@ data OpenMiningRoundSummary = OpenMiningRoundSummary with
     totalUnfeaturedAppRewardCoupons : Decimal
     totalSvRewardWeight : Int
     optTotalValidatorFaucetCoupons : Optional Int
+    optTotalValidatorLivenessActivityRecords : Optional Decimal -- ^ Replaces optTotalValidatorFaucetCoupons
```

Comment:
@meiersi-da So decimal weights are a bit more annoying than I though. I guess lesson learned: Avoid using Ints where you don't actually care about it being an Integer.

I am slightly concerned about what happens at the point of switchover where different SVs may create different confirmations for StartIssuing. My impression is that this is fine:

Worst case the StartIssuing trigger will create both for one SV. ExecuteConfirmedAction identities things by action so those are just different actions. We can only get one of the two through and the other is stale and will eventually get cleaned up through expiry.

Reply:
> @meiersi-da So decimal weights are a bit more annoying than I though. I guess lesson learned: Avoid using Ints where you don't actually care about it being an Integer.

Yeah! I'm also unhappy about my BPS choice for SV weights. Should have just used a `Decimal` there too.

> I am slightly concerned about what happens at the point of switchover where different SVs may create different confirmations for StartIssuing. My impression is that this is fine:
> 
> Worst case the StartIssuing trigger will create both for one SV. ExecuteConfirmedAction identities things by action so those are just different actions. We can only get one of the two through and the other is stale and will eventually get cleaned up through expiry.

Yes, I agree. I expect this to work fine.